include/rocksdb/options.h

```diff
@@ -951,10 +951,13 @@ struct DBOptions {
   enum AccessHint { NONE, NORMAL, SEQUENTIAL, WILLNEED };
   AccessHint access_hint_on_compaction_start = NORMAL;
 
-  // If non-zero, we perform bigger reads when doing compaction. If you're
+  // The size RocksDB uses to perform readahead during compaction read.
+  // If set zero, RocksDB will sanitize it to be 2MB during db open.
+  // If you're
```

Comment:
I guess the reason for not just changing the default to 2MB is because we never want compaction_readahead_size to be 0?

Reply:
I didn't make the default value change in this PR mainly to minimize the change (just comment change) to be cherry-picked to 8.6.   Yes we can consider changing the default to 2MB too in the future